nixos/modules/services/networking/sickbeard.nix

```diff
@@ -0,0 +1,79 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.sickbeard;
+  sickbeard = cfg.package;
+in
+{
+
+  ###### interface
+
+  options = {
+    services.sickbeard = {
+      enable = mkOption {
+        default = false;
+        description = "Whether to enable the sickbeard server.";
+      };
+      package = mkOption {
+        type = types.package;
+        default = pkgs.sickbeard;
+        example = literalExample "pkgs.sickrage";
+        description =''
+          Enable <literal>pkgs.sickrage</literal> or <literal>pkgs.sickgear</literal>
+          as an alternative to SickBeard
+        '';
+      };
+      dataDir = mkOption {
+        default = "/var/lib/sickbeard";
+        description = "Path where to store data files.";
+      };
+      configFile = mkOption {
+        default = "/var/lib/sickbeard/config.ini";
```

Comment:
Should be "${dataDir}/config.ini and be sure to add a rec above so you can reference dataDir properly, I believe.